CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ githubr:::.unitTests()
 ```
 
 ### Thank you section!
-Many people have provided invaluable feedback in the development of the rGithubClient - can't wait to add more! Here is a rolling list:
+Many people have provided invaluable feedback in the development of the githubr client - can't wait to add more! Here is a rolling list:
```

Comment:
That's what I was going for :)

![Sneaky](https://media3.giphy.com/media/BiySQOIYzJfPy/200.gif)